package.json

```diff
@@ -8,9 +8,9 @@
     "start:test": "npm run eslint && node_modules/gulp/bin/gulp.js start-serverless",
     "end:test": "node_modules/gulp/bin/gulp.js kill-serverless",
     "debug": "export SLS_DEBUG=* && node --inspect ./node_modules/serverless/bin/serverless offline start",
-    "test": "node_modules/.bin/mocha tests/**/*.unitTest.js",
+    "test": "./node_modules/.bin/mocha tests/**/*.unitTest.js",
```

Comment:
was this not working previously? seems you had to explicitly change the dir where its looking for test.

Reply:
Accidentally preserved change which makes no difference - changes path to Mocha, not to tests themselves, but both syntaxes are used interchangeably within even a single project.json. Can remove if concerned.